distributions/Jamulus.1

```diff
@@ -60,82 +60,82 @@ is also possible.
 .Pp
 One participant starts
 .Nm
-in server mode, ideally on a dedicated server (virtual) machine;
-all participants start the (graphical) client which transmits audio
-to the server, receiving back a mixed stream.
+in Server mode, ideally on a dedicated Server (virtual) machine;
+all participants start the (graphical) Client which transmits audio
+to the Server, receiving back a mixed stream.
 Use of a metronome is recommended.
 Clients should be connected using ethernet, not wireless, and use
 proper headphone and microphone connections, not Bluetooth.
-The server should run on a low-latency system, ideally not a VM.
+The Server should run on a low-latency system, ideally not a VM.
 .Pp
 Running
 .Nm
-without any extra options launches the full graphical client.
+without any extra options launches the full graphical Client.
 .Pp
 The options are as follows:
```

Comment:
The options are all a bit mixed up here.

In the `--help` output, they're now "Common" first, then "Server only", followed by "Client only".  It might be easier if the man page were the same.

Reply:
Last we spoke about this man page business, we hoped we'd automate it all out of `-h` (dropping all the stuff about recommending metromes and whatnot in the process) so that we don't manually maintain things in three places. It's only really there to satisfy Debian. Also doesn't help that I don't understand the syntax.

Reply:
@pljones options in manpages are supposed to be alphabetically sorted, case-insensitive (uppercase `-Y` before lowercase `-y` but after lowercase `-x`). Doing it a different way is very much **not** helpful.

@gilgongo manpages are supposed to be _much_ more than just an option summary (you can get that from `--help`). It should also precisely explain how to use, concepts necessary, etc.

Reply:
> @pljones options in manpages are supposed to be alphabetically sorted, case-insensitive (uppercase `-Y` before lowercase `-y` but after lowercase `-x`). Doing it a different way is very much **not** helpful.

They are alphabetical.  They're just in separate sections that aren't related.